proto/buf/alpha/module/v1alpha1/module.proto

```diff
@@ -20,6 +20,27 @@ import "buf/alpha/breaking/v1/config.proto";
 import "buf/alpha/lint/v1/config.proto";
 import "google/protobuf/timestamp.proto";
 
+enum DigestType {
+  DIGEST_TYPE_UNSPECIFIED = 0;
+  DIGEST_TYPE_SHAKE256 = 1;
+}
+
+// Hash represents a hash function's value.
+message Hash {
```

Comment:
s/Hash/Digest (for consistency with corresponding types in manifest package)

Also, I asked in previous comment but didn't get an answer: why have two separate representations (this one, generated from proto, and a hand-written version in pkg/manifest)? Seems like you could just use the generated proto messages and enums and not have to maintain hand-written types.